source/codegen/metadata/nidmm/functions.py

```diff
@@ -1495,6 +1616,27 @@
         ],
         'returns': 'ViStatus'
     },
+    "InitExtCal": {
+        "init_method" : True,
+         "parameters": [
+         {
+             "name": "resourceName",
+             "direction": "in",
+             "type": "ViRsrc"
+         },
+         {
+             "name": "calibrationPassword",
+             "direction": "in",
+             "type": "ViString"
```

Comment:
I have used ViString for all the parameters that say Password. The chm file says ViChar[ ], but I wont be able to auto generate the API implementation without the 'size' mentioned for this array.
![image](https://user-images.githubusercontent.com/47028785/115587393-45faa300-a2eb-11eb-8359-02aef921dd6d.png)
Another option would be to use 'fixed' mechanism and set the size to 8 characters as mentioned.
In this case I felt ViString would be a good option, correct me if I'm wrong @jonlugoatx @reckenro

Reply:
The fact that it mentions a termination character means that the size need not be fixed nor specified by another parameter, so we shouldn't use a size mechanism here. I see no reason why a type of `ViChar[]` as an input parameter shouldn't be treated any differently than `ViString` by our code generator. If it's easy to update the generator to allow this, then go for it. Otherwise, I'm okay with this being defined as `ViString`.

Reply:
I will leave it as ViString itself